src/qaqc/download_and_flag_fxns/kw_known_probs.R

```diff
@@ -0,0 +1,10 @@
+# KW's field notes flags
+#finding instances in HydroVu and in the field notes where we identified issues with certain site-parameters.
+
+add_malfunction_flag <- function(df){
+
+  df <- df %>%
+    add_flag((year == "2022" & DT_round <= "2022-05-25 15:00" & site == "timberline" & parameter == "DO"), "sensor malfunction") %>%
```

Comment:
question: will we eventually want to automate this?

Reply:
In an ideal scenario, any instance of sensor malfunction would be ID'd through the flags we already have. But! Yes, I think that it would be great to develop an automated way of doing this. Maybe a table where field techs add start date and date, site, and parameter(s) affected that would be fed into this flag function.